platform/ios/src/MGLCalloutView.h

```diff
@@ -46,6 +46,22 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)dismissCalloutAnimated:(BOOL)animated;
 
+@optional
+
+/**
+ A Boolean value indicating whether the callout view should be anchored to
+ the corresponding annotation. The exact position can be adjusted by 
+ overriding -[UIView setCenter:]
```

Comment:
Good question. As of https://github.com/mapbox/mapbox-gl-native/pull/7646/commits/46dd0a6c402df4959fb6a20de4f874d6cd48bd47 , it defaults to `dismissesAutomatically`.